llvm/lib/Target/DirectX/DXILOpLowering.cpp

```diff
@@ -22,9 +22,12 @@
 #include "llvm/IR/IntrinsicsDirectX.h"
 #include "llvm/IR/Module.h"
 #include "llvm/IR/PassManager.h"
+#include "llvm/MC/TargetRegistry.h"
 #include "llvm/Pass.h"
 #include "llvm/Support/ErrorHandling.h"
 
+#include "iostream"
```

Comment:
We do not use iostream in llvm: https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden

Reply:
> We do not use iostream in llvm: https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden

Deleted.